config/crd/bases/kepler.system.sustainable.computing.io_keplers.yaml

```diff
@@ -81,91 +84,85 @@ spec:
                         default: true
                         type: boolean
                     type: object
-                  modelPath:
-                    default: ""
-                    type: string
-                  modelServerRequiredPath:
-                    default: ""
-                    type: string
-                  modelServerURL:
+                  url:
                     default: ""
                     type: string
-                  port:
-                    default: 8100
-                    type: integer
-                  promServer:
-                    default: ""
-                    type: string
-                type: object
-              model-server-features:
-                properties:
-                  includePVandPVCFinalizer:
-                    default: false
-                    type: boolean
                 type: object
             type: object
           status:
             description: KeplerStatus defines the observed state of Kepler
             properties:
               conditions:
```

Comment:
Is it a good time to make `conditions` an array to align with existing practices?

Reply:
This refactor will include PI changes so the plan is to include all changes to API in this release. Also the condition should be an array type as per the specification. This was the reason why the console was breaking because it expected an array type and was receiving an object.

Reply:
Indeed it is, as we are breaking the API anyway.